libr/include/r_crypto.h

```diff
@@ -98,6 +99,12 @@ extern RCryptoPlugin r_crypto_plugin_serpent;
 #define R_CRYPTO_SERPENT 1<<12
 #define R_CRYPTO_ALL 0xFFFF
 
+#define R_ENCODER_NONE 0
+#define R_ENCODER_B64 1
```

Comment:
use 1ULL or (ut64)1 to enforce 64bit size here. if we used an enum that coul dbe a problem for some compilers. so better stay with defines. about the name.. this will be used also for decoding. so what about R_CODEC ?